src/parser/resources/item/item_parser.py

```diff
@@ -101,12 +102,18 @@ def run(self):
             else:
                 is_disabled = False
             parsed_item_data['Disabled'] = is_disabled
-                
 
             all_items[key] = parsed_item_data
 
         chart = MermaidDiagram(title='Items', nodes=self.nodes, links=self.links)
 
+        # Create and save Item objects
+        Item.hashToObjs(all_items)
+        Item.saveObjects(free_memory=True)
+
+        #Item.loadObjects() #if commented out, the following line will throw an error
+        print(Item.objects['armor_upgrade_t4'].getProp('Cost'))
```

Comment:
Should this be commented out as well?

Reply:
> Should this be commented out as well?

I left it uncommented in the draft before I got back to it today so you could see the error, but yes in order to run you would either make #115 a comment or uncomment #114